WordPress/gutenberg
# [Mobile] Disable the space-between justify option for buttons block

Fixes: https://github.com/WordPress/gutenberg/issues/30560

**Related PR:**
`gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3356

## Description

This PR disables the space-between option for the buttons block on mobile. This partially reverts #29160, since we do not have support for this yet on mobile. This can be re-added once it has support and has been tested.

## How has this been tested?

* Add a button(s) block in a new post
* Tap the justify settings
* Observe that there is no "space-between" option listed

## Checklist:
